content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md

```diff
@@ -59,8 +59,19 @@ Autoscaler. The monitoring pipeline fetches metrics from the kubelet and
 then exposes them to Kubernetes via an adapter by implementing either the
 `custom.metrics.k8s.io` or `external.metrics.k8s.io` API. 
 
-[Prometheus](https://prometheus.io), a CNCF project, can natively monitor Kubernetes, nodes, and Prometheus itself.
-Full metrics pipeline projects that are not part of the CNCF are outside the scope of Kubernetes documentation.  
+Kubernetes is designed to work with [OpenMetrics](https://openmetrics.io/), 
+which is one of the
+[CNCF Observability and Analysis - Monitoring Projects](https://landscape.cncf.io/card-mode?category=monitoring&project=graduated,incubating,member,no&grouping=category&sort=stars),
+built upon and carefully extending [Prometheus exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/)
+in almost 100% backwards-compatible ways.
+
+If you glance over at 
+[CNCF Landscape](https://landscape.cncf.io/card-mode?category=monitoring&project=graduated,incubating,member,no&grouping=category&sort=stars), 
+you can observe that the Kubernetes project doesn't provide a recommendation 
```

Comment:
```suggestion
you can observe that the Kubernetes project doesn't provide a recommendation 
```
just a quick question or more like a doubt, how can you say that k8s don't provide a recommendation of a set of projects **just by looking at the landscape**? There is no set of lines that gives this kind of proof. Maybe we can modify this!

Reply:
Maybe because when we look at CNCF landscape there are tons of tools to work with k8s, not only Prometheus. One can check out other tools and work with one they think suits them best.

Reply:
![image](https://user-images.githubusercontent.com/101946115/216019003-a16fd1be-086a-4194-ba14-5c9b766513c1.png)

Like if I'm looking over at Monitoring, I can see there are many projects other than Prometheus, and one can check out these and work with the one they think suits them best like Open Metrics or Thanos for example